src/saga/adaptors/context/myproxy.py

```diff
@@ -135,7 +148,7 @@ def _initialize (self, session) :
             cmd += " --proxy_lifetime %s"  %  api.life_time
 
         # store the proxy in a private location
-        proxy_store    = "%s/.saga/proxies/"   %  os.environ['HOME']
+        proxy_store    = "%s"   %  self.base_work_dir
```

Comment:
Oups another mistake in my PR, I didn't mean to submit this file, as I have (for now) no use of proxies configuration, anyway I'll do modification for default value.